src/material/tabs/tab-group.ts

```diff
@@ -566,6 +566,10 @@ export class MatTabGroup extends _MatTabGroupBase {
       defaultConfig && defaultConfig.fitInkBarToContent != null
         ? defaultConfig.fitInkBarToContent
         : false;
+    this.stretchTabs =
```

Comment:
This should default to `true` on the MDC-based tabs and `false` on the legacy ones.

Reply:
Thanks for catching that. Updated

Reply:
I am guessing all I needed to do was change the default value from `false` to `true` in this case as  `_stretchTabs` is set to `true` above as a default [here](https://github.com/angular/components/blob/3c53deb86477a43cedf930acc6ed7105147006e0/src/material/tabs/tab-group.ts#L556)

Is that correct?